<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget{
background-color:white;
}</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="saveFileButton">
     <property name="text">
      <string>Сохранить файл</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Дата                След. день            Пред. день      До след.  дня списка            Високосный год  </string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="2">
      <widget class="QSpinBox" name="assertYearSpinBox"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="assertDaySpinBox"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="assertDayLabel">
       <property name="text">
        <string>Введите день</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="assertLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="assertConfirmButton">
       <property name="text">
        <string>Подтвердить</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="assertMonthSpinBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="assertMonthLabel">
       <property name="text">
        <string>Введите месяц</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="assertYearLabel">
       <property name="text">
        <string>Введите год</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="chooseFile">
     <property name="text">
      <string>Выбрать файл</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="6" column="3">
      <widget class="QPushButton" name="editDateButton">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Колличество дней до введенной даты</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="birthdayButton">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="weekNumberButton">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Колличество дней до ближайшего дня рождения</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="addDateButton">
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Номер недели в году</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="durationLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="weekNumberLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Добавить дату в список</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Изменить выбранную дату</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="birthdayLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="durationButton">
       <property name="text">
        <string>Рассчитать</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="removeDateButton">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Удалить выбранную дату из списка</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
